apps/roboshield/src/pages/about.tsx

```diff
@@ -0,0 +1,26 @@
+import { getPageServerSideProps } from "@/roboshield/lib/data";
```

Comment:
Shouldn't this be one of the catch all pages `....tsx` or `[...].tsx`? I'm assuming all pages will just call render blocks

Reply:
@kilemensi Since we only have 2 pages, I think there is no need for a catch-all page. Perhaps we can introduce it when there's an actual need for any extra page

Reply:
Isn't `pages/index.tsx` going to be exact copy of this then @kelvinkipruto ?